README.md

```diff
@@ -1,75 +1,56 @@
-
-### Spares
-
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
-
-### 10th frame
-
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
-
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
-
-### Gutter Game
-
-A Gutter Game is when the player never hits a pin (20 zero scores).
-
-### Perfect Game
-
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
-
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
-
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
-
-## Code Review
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Note that referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want.
+## Bowling challenge
+
+#### User stories
+```
+As a player
+To know how far along in the game we are
+I want to know what frame we are at
+```
+```
+As a player
+To know how many rolls I have left
+I want to know what roll we are at
+```
+```
+As a player
+To see my score after a roll
+I want to see the amount of pins knocked down per roll
+```
+```
+As a player
+To see my score after a frame
+I want to see the points per frame added up together
+```
+```
+As a player
+To see my score throughout the game
+I want to see my total points 
+```
+```
+As a player
+To understand my bonusses
+I want to see the bonus points explaned to me
+```
+```
+As a player
+To finish the game
+I want to get an extra roll if I hit 10 pins in the last frame
+```
+
+#### special rules:
+
+- Strike: All 10 pins in 1 roll.
+    Ends frame
+    bonus = number of pins down in the next two rolls. - That is either a frame or an other strike.
+
+- Spare: All 10 pins in 2 rolls (1 frame).
+    bonus = number of pins in the next roll
+
+- 10th Frame: Player gets spare or strike in the last frame
+    bonus = the last roll
+    never more than 3 rolls
+
+- Gutter game: No pins hit at all
+
+- Perfect game: 12 strikes
+    300 points
```

Comment:
Would be nice to have a quick list of console commands to run the logic to see it in action, but you have covering tests so it's not too big of a deal!